### ISSUE TITLE
Support for NOT ILIKE

### ISSUE THREAD

**Issue by klauck** [hyrise/sql-parser#182]
Inspecting the rail road diagram, I noticed that we support ILIKE but not NOT ILIKE.
ILIKE can be used instead of LIKE to make the operator case insensitive.
ILIKE is not in the SQL standard, but I think we should support both (ILIKe and NOT ILIKE) or none.

**Comment by cdmh** [comment]
I think `NOT` should be supported in front of any expression, and  to specific to `LIKE`/`NOT LIKE` keywords, shouldn’t it?

**Comment by klauck** [comment]
Yes, ```NOT``` should and is already supported in front of any expression as unary operator.
However, ```ILIKE``` requires a value/attribute before and pattern after, and is, thus handled separately in the parser.
